src/main/java/com/meetme/plugins/jira/gerrit/tabpanel/GerritReviewsTabPanel.java

```diff
@@ -94,6 +96,9 @@ public ShowPanelReply showPanel(ShowPanelRequest arg0) {
             isShowing = false;
         }
 
+        if (configuration.getUseGerritProjectWhitelist() && ! isGerritProject(arg0.issue()))
+            isShowing = false;
```

Comment:
Please follow surrounding code styles:  braces around multi-line if blocks, and no space after `!`

```java
        if (configuration.getUseGerritProjectWhitelist() && !isGerritProject(arg0.issue())) {
            isShowing = false;
        }
```